docs/adr/0006-url-encode-path-parameters.md

```diff
@@ -0,0 +1,51 @@
+# 0006 - URL-encode path parameters that contain forward slashes
+
+2023-03-16
+
+## Status
+
+Accepted
+
+## Context
+
+Users must be able to search the system for nominals on Police National Computer (PNC) ID. PNC ID is a unique person
```

Comment:
> Users must be able to search the system for nominals on Police National Computer (PNC) ID. PNC ID is a unique person

'nominals' is definitely a word I've heard MAPPS use a lot. I'm not sure whether by using it here we'd be gearing our documentation directly at them rather than a generic consumer of our API. I've asked MoJ what their preferred term is for this.

Reply:
Had a chat with Martin, they would prefer us avoiding this term unless it's directly requested. The preferred alternatives are:
- Person
- Person in Prison
- Person on Probation